pacman/operations/placer_algorithms/one_to_one_placer.py

```diff
@@ -88,31 +89,49 @@ class OneToOnePlacer(RadialPlacer):
 
     def __call__(self, machine_graph, machine):
 
+        # Iterate over vertices and generate placements
+        progress = ProgressBar(
+            machine_graph.n_vertices + 3, "Placing graph vertices")
```

Comment:
magic number 3. can we at least have a comment saying what these magic 3 steps are.

Reply:
comment added